exec/java-exec/src/main/java/org/apache/drill/exec/physical/impl/xsort/managed/SortConfig.java

```diff
@@ -71,8 +72,8 @@
 
   private final int mSortBatchSize;
 
-  public SortConfig(DrillConfig config) {
-
+  public SortConfig(FragmentContext context) {
+    DrillConfig config = context.getConfig();
```

Comment:
Suggestion: pass in the original `DrillConfig` plus an option manager rather than the fragment context. The suggestion minimizes undesired dependencies.